### PR TITLE
use yum-qemu-ev instead of base for setting qemu-kvm

### PR DESCRIPTION
**Pull request by knightsamar**

### CODE REVIEW

File: recipes/packer_pipeline_node.rb [hunk 1]
```diff
@@ -102,7 +102,7 @@
 end
 
 # setup qemu so that we can build images!
-include_recipe 'base::kvm' if node['kernel']['machine'] == 'x86_64'
+include_recipe 'yum-qemu-ev' if node['kernel']['machine'] == 'x86_64'
```
Comment (ramereth) [inline recipes/packer_pipeline_node.rb]: Two things:

- [ ] You need to keep ``base::kvm`` but move it AFTER ``yum-qemu-ev``
- [ ] Remove the architecture statement, this should work on ppc64le the same.

Comment (knightsamar) [inline recipes/packer_pipeline_node.rb]: @ramereth  Done!

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.8.9.  Have a nice day!

**Pull request merged**